python/pfs/utils/makePfsDesign.py

```diff
@@ -0,0 +1,110 @@
+import numpy as np
+from pfs.datamodel import PfsDesign, FiberStatus, TargetType
+from pfs.datamodel.utils import calculate_pfsDesignId
+
+from pfs.utils.fiberids import FiberIds
+
+
+def makePfsDesignFile(x, y,
+                      raBoresight=100, decBoresight=100, posAng=0, arms='br',
+                      tract=1, patch='1,1', ra=None, dec=None, catId=-1, objId=-1, targetType=TargetType.SCIENCE,
+                      fiberStatus=FiberStatus.GOOD,
+                      fiberFlux=np.NaN, psfFlux=np.NaN, totalFlux=np.NaN,
+                      fiberFluxErr=np.NaN, psfFluxErr=np.NaN, totalFluxErr=np.NaN,
+                      filterNames=None, guideStars=None):
+    """ Make PfsDesign object from cobra x and y required positions.
+
+        Parameters
+        ----------
+        x : array_like
+            cobra x position (1..2394).
+        y: array_like
+            cobra y position (1..2394).
```

Comment:
Yes, I meant an array of 2394 values, my bad.
I think it was more natural for us because we were asking position from dots.x dots.y.
But yes, I can change that to an array of (2394, 2) to make it consistent with datamodel constructor.